root/static/less/nav.less

```diff
@@ -1,3 +1,13 @@
+// For some reason in Safari, the .clearfix() styling
+// is causing the icons to be outside of the viewport.
+.nav {
+  &:before,
+  &:after {
+    content: "" !important;
+    display: block !important;
+  }
+}
```

Comment:
I'm still getting used to using the dev tools on a mobile device, but this seemed to do the trick.